test/integration/scheduler_test.go

```diff
@@ -404,3 +404,177 @@ func makeNPods(c client.Interface, N int) {
 	}
 	wg.Wait()
 }
+
+func TestMultiScheduler(t *testing.T) {
+	framework.DeleteAllEtcdKeys()
```

Comment:
This test is too long.
We can improve it by changing to use table test.

How about the following workflow:
- create two schedulers
- craft tables of pods of a variety of combinations
  - default, "foo", "non-exists", combinatory ones (do we support it?)